climt/_core/initialization.py

```diff
@@ -151,9 +151,14 @@ class PressureFunctionDiagnosticComponent(DiagnosticComponent):
 
     input_properties = {
         'air_pressure': {
-            'dims': ['*'],
```

Comment:
The dims of air pressure and surface air pressure should be different. You probably want to change this to `['mid_levels', '*']`.